File: PYTHON/freq-101.py
num=input().split(" ")
n=int(num[0])
m=int(num[1])
arr=[]
temp=input().split(" ")
for i in range(n):
    arr.append(int(temp[i]))
count=0
for j in range(n):
    if(arr[j]==m):
        count+=1
print(count)

File: Extra/freq-101.js
let fs = require("fs");
let data = fs.readFileSync("C:/Users/ASUS/OneDrive/Desktop/10x/input.txt", 'utf-8');
let idx = 0;
data = data.split('\n');

function readLine() {
    idx++;
    return data[idx - 1].trim();
}
let num = readLine().split(" ");
let n = parseInt(num[0]);
let k = parseInt(num[1]);
let arr = [];
let temp = readLine().split(" ");
for (let i = 0; i < n; i++) {
    arr.push(parseInt(temp[i]));
}
let count = 0;
for (let j = 0; j < n; j++) {
    if (arr[j] == k) {
        count++;
    }
}
console.log(count);